packages/account/src/Components/poi/idv-status/idv-failed/idv-failed.tsx

```diff
@@ -361,79 +372,77 @@ const IdvFailed = ({
+            ? localize('Verify')
+            : localize('Update profile');
+
     return (
         <Formik
             initialValues={rest_state?.form_initial_values ?? {}}
             onSubmit={onSubmit}
             validate={validateFields}
             className='proof-of-identity__container'
         >
-            {({ isSubmitting, isValid, dirty, status, values, errors }) => (
+            {({ isSubmitting, isValid, dirty, status, values }) => (
                 <Form
                     className={classNames('proof-of-identity__mismatch-container', {
                         'upload-layout': is_document_upload_required,
+                        'min-height': values?.document_type?.id === IDV_NOT_APPLICABLE_OPTION.id,
                     })}
                 >
                     <FormBody className='form-body' scroll_offset={setScrollOffset()}>
-                        <Text size={isMobile() ? 'xs' : 's'} weight='bold' align='center'>
+                        <Text size={is_mobile ? 'xs' : 's'} weight='bold' align='center'>
                             <Localize i18n_default_text='Your identity verification failed because:' />
                         </Text>
                         {(status?.error_msg || idv_failure?.failure_message) && (
                             <HintBox
                                 className={classNames('proof-of-identity__failed-message', 'hint-box-layout')}
                                 icon='IcAlertDanger'
                                 message={
-                                    <Text as='p' size={isMobile() ? 'xxs' : 'xs'} data-testid={mismatch_status}>
+                                    <Text as='p' size={is_mobile ? 'xxs' : 'xs'} data-testid={mismatch_status}>
                                         {status?.error_msg ?? idv_failure?.failure_message}
                                     </Text>
                                 }
                                 is_danger
                             />
                         )}
                         {is_document_upload_required && (
-                            <React.Fragment>
-                                <Text size='xs' align={isMobile() ? 'left' : 'center'}>
+                            <div>
+                                <Text size='xs' align={is_mobile ? 'left' : 'center'}>
                                     <Localize i18n_default_text='Let’s try again. Choose another document and enter the corresponding details.' />
                                 </Text>
                                 <FormSubHeader title={localize('Identity verification')} />
                                 <IDVForm selected_country={chosen_country} class_name='idv-layout idv-resubmit' />
-                                <FormSubHeader title={localize('Details')} />
-                            </React.Fragment>
+                                {values?.document_type?.id !== IDV_NOT_APPLICABLE_OPTION.id && (
```

Comment:
We are checking whether values?.document_type?.id !== IDV_NOT_APPLICABLE_OPTION.id and values?.document_type?.id === IDV_NOT_APPLICABLE_OPTION.id in many places in this file. Can we make a const for this and reuse it?

Reply:
yes, but the values are used inside the Formik scope.
I will try to implement reusable function for this 👍